### PR TITLE
Analysis Menu-Cursor changes Issue #124

### PR DESCRIPTION
**Pull request by sergiodmn**
Changes proposed by Maijin to improve analysis options.

### CODE REVIEW

File: src/optionsdialog.cpp [hunk 1]
```diff
@@ -191,15 +229,13 @@ QString OptionsDialog::analysisDescription(int level)
...
     {
     case 0:
-        return tr("-");
+        return tr("No analysis");
     case 1:
-        return tr("Minimum");
+        return tr("Auto-Analisis (aaa)");
     case 2:
-        return tr("Basic");
+        return tr("Auto-Analysis Experimental (aaaa)");
```
Comment (Maijin) [inline src/optionsdialog.cpp]: I would remove (aaaa) and (aaa)

File: src/optionsdialog.cpp [hunk 2]
```diff
@@ -191,15 +229,13 @@ QString OptionsDialog::analysisDescription(int level)
     switch (level)
     {
     case 0:
-        return tr("-");
+        return tr("No analysis");
     case 1:
-        return tr("Minimum");
+        return tr("Auto-Analisis (aaa)");
```
Comment (Maijin) [inline src/optionsdialog.cpp]: Typo Analysis

File: src/optionsdialog.cpp (hunk 1 above)
Comment (sergiodmn) [reply src/optionsdialog.cpp]: I left the (aaaa) and (aaa) to help people understand the kind of analysis that is being made in r2, so if anyone wants to use the console, they know which command to use.

**Comment by radare** [comment]
lgtm

**Comment by hteso** [comment]
I'll check it ASAP :)

**Comment by hteso** [comment]
Seems to work, but the UX of the OptionsDialog needs lots of love. I'll improve that before accepting the PR :) Thanks for the work!

**Comment by radare** [comment]
needs to be rebased. any update @hteso ?

**Comment by hteso** [comment]
I'm working on it, I'm not having that much time lately. I'll have it ASAP

**Pull request closed without merging**